src/opm/input/eclipse/EclipseState/Grid/NNC.cpp

```diff
@@ -171,14 +175,127 @@ bool is_neighbor(const EclipseGrid& grid, std::size_t g1, std::size_t g2) {
         }
     }
 
+    void NNC::load_editr(const EclipseGrid& grid, const Deck& deck) {
+        using NNCinsert = std::tuple<std::array<std::size_t,3>,double>;
+        std::vector<NNCinsert> nnc_editr;
+        std::size_t insertion_index = 0;
```

Comment:
Mmh. because of `size_t`? Should be implicitly included as this is by far not the only place where it is used. Added it anyway.

Reply:
indeed, because of std::size_t. i know that it was obviously implicitly available, but we should be explicit, we don't want to be at the mercy of libstdc++ header reshuffles etc.